compile_epub.sh

```diff
@@ -1,10 +1,27 @@
 #!/usr/bin/zsh
-for f in `seq 1 $1`; do
-  echo "compiling Practical Guide to Evil - Book $f"
-  pandoc -o "A Practical Guide to Evil - Book $f.epub" "A Practical Guide to Evil - Book $f.md"
-done
-
-for f in `seq 1 $2`; do
-  echo "compiling Pale Lights - Book $f"
-  pandoc -o "Pale Lights - Book $f.epub" "Pale Lights - Book $f.md"
-done
\ No newline at end of file
+
+if [ -z "$1" ]; then
+  echo "Usage: ./compile_epub.sh <First x Practical Guide to Evil Books> <First x Pale Lights Books>"
+fi
+
+if [ $1 -ge 0 ]; then
+  for f in `seq 1 $1`; do
+    echo "compiling Practical Guide to Evil - Book $f"
+    pandoc -o "output/A Practical Guide to Evil - Book $f.epub" "output/A Practical Guide to Evil - Book $f.md"
+  done
+fi
+
+if [ $2 -ge 0 ]; then
+  for f in `seq 1 $2`; do
+    echo "compiling Pale Lights - Book $f"
+    pandoc -o "output/Pale Lights - Book $f.epub" "output/Pale Lights - Book $f.md"
+  done
+
+  # Really hacky but don't know how else to 'automate' this
```

Comment:
up to you, but we could also do this in a Python script, and get rid of the bash script completely?

Reply:
Yeah sure, I will change it tomorrow. What do you think about moving the .md and .epub files into their own output directory? Because with the split of the Books the root directory is getting pretty cluttered...

Reply:
No strong feelings :smile: I'd like the full md/epub files to end up in the repo still but apart from that I don't really mind :smile: